### PR TITLE
Add subject strisnegative

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: subjects/strisnegative/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Expected function
+```go
+func StrisNegative(str string) {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function :
```
Comment (Hamzaelkhatri) [inline subjects/strisnegative/README.md:19]: no need for this line

File: subjects/strisnegative/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+```go
+package main
+
+import "piscine"
+
+func main() {
+	piscine.StrisNegative("585")
+	piscine.StrisNegative("-58")
+	piscine.StrisNegative("55s44")
```
Comment (Hamzaelkhatri) [inline subjects/strisnegative/README.md:29]: add 
```go
piscine.StrisNegative("101-1331")
piscine.StrisNegative("5544-")
```

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: subjects/strisnegative/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+### Usage
+
+Here is a possible program to test your function :
+
+```go
+package main
+
+import "piscine"
```
Comment (Hamzaelkhatri) [inline subjects/strisnegative/README.md:24]: remove 'piscine` header

File: subjects/strisnegative/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+func main() {
+	piscine.StrisNegative("585")
```
Comment (Hamzaelkhatri) [inline subjects/strisnegative/README.md:27]: the same here

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]

File: subjects/strisnegative/README.md (hunk 4 above)
Comment (zainabdnaya) [reply subjects/strisnegative/README.md]: Done

File: subjects/strisnegative/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+}
+```
+
+### Usage
+
+Here is a possible program to test your function :
+
+```go
+package main
+
```
Comment (MSilva95) [inline subjects/strisnegative/README.md:23]: be careful with the format... there are extra lines here and this ones will be visible in the readme.

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/strisnegative/README.md (hunk 5 above)
Comment (zainabdnaya) [reply subjects/strisnegative/README.md:23]: corrected

File: subjects/strisnegative/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+## isnegative
```
Comment (MSilva95) [inline subjects/strisnegative/README.md:1]: make the same name as the folder and func

File: subjects/strisnegative/README.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+Write a function named `IsNegative()` that defines if a number (You should check if it's a number) is negative or positive.
```
Comment (MSilva95) [inline subjects/strisnegative/README.md:3]: Write a function called `IsNegative()` that defines whether a number is negative or positive.
- Your function prints `P` if the number is positive
- Your function prints `F` if the number is negative
- If the number is zero, print `0`.
- If it's not a number, print `('\n') `.
- Your program should always print `('\n') ` at the end of the output.

**Review by MSilva95 — CHANGES_REQUESTED** [review]

**Review by MSilva95 — APPROVED** [review]

**Review by Hamzaelkhatri — APPROVED** [review]

**Pull request merged**